e2e/cypress/tests/integration/account_settings/profile/profile_picture_spec.ts

```diff
@@ -89,4 +78,54 @@ describe('Profile > Profile Settings > Profile Picture', () => {
             should('be.visible').
             and('contain', 'Image limits check failed. Resolution is too high.');
     });
+
+    it('MM-T2078 Profile picture: file type PNG accepted', () => {
```

Comment:
I believe each of these tests below needs different test number (i.e.  MM-XXX). @saturninoabril please correct me if I am wrong here.